src/site/content/en/blog/introducing-model-viewer/index.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: 'Announcing <model-viewer> 1.0'
+subhead: 3D models for your web page as easily as writing HTML.
+authors:
+  - rkochman
+  - joemedley
+description: |
+3D models are more relevant than ever. Retailers bring in-store shopping
+experiences to customers' homes. Museums let anyone see artifacts from anywhere.
+3D is difficult to do without a deep knowledge of 3D technologies or third-party
+hosting. <model-viewer> 1.0 makes these tasks as easy as writing HTML.
```

Comment:
"as adding an HTML tag" (or element)? It feels weird to say that writing HTML is easy. I mean, it doesn't sound quite right.